datahub-web-react/src/app/ingest/source/executions/useRefreshIngestionData.ts

```diff
@@ -0,0 +1,18 @@
+import { useState, useEffect } from 'react';
+
+export default function useRefreshIngestionData(refresh: () => void, hasActiveExecution: () => boolean) {
+    const [refreshInterval, setRefreshInterval] = useState<NodeJS.Timeout | null>(null);
```

Comment:
I'm suprised to see the type `NodeJS.Timeout` given this is running on the browser

Reply:
Yeah that's an interesting type